sermant-plugins/sermant-mq-consume-prohibition/rocketmq-plugin/src/main/java/com/huaweicloud/sermant/mq/prohibition/rocketmq/utils/PullConsumerLocalInfoUtils.java

```diff
@@ -84,16 +84,16 @@ public static Collection<MessageQueue> getMessageQueue() {
     }
 
     /**
-     * 设置消费者assign的消费队列
+     * Set up a consumption queue for consumer assignment
```

Comment:
Set up consumption queues for consumer assignment

Reply:
done